space-framework/pkg/msclientlib/client.go

```diff
@@ -42,14 +42,15 @@ type KubestellarSpaceInterface interface {
 	//Space(name string, namespace ...string) (client mcclientset.Interface)
 
 	// ConfigForSpace returns rest config for given space.
-	ConfigForSpace(name string, namespace ...string) (*rest.Config, error)
+	ConfigForSpace(name string, providerNS string) (*rest.Config, error)
```

Comment:
The comment on this method needs to tell the user how to get the default provider.  Is it with `providerNS=="default"` ?